frontend/src/App.tsx

```diff
@@ -447,6 +447,14 @@ export class App extends PureComponent<Props, State> {
           MetricsManager.current.getAndResetDeltaCounter()
         )
 
+        const { availableThemes, activeTheme } = this.props.theme
+        const customThemeDefined =
+          availableThemes.length > createPresetThemes().length
```

Comment:
Why is there a newline here?

Reply:
The autoformatter probably decided to add a newline due to line length being too long